### PR TITLE
wakeonlan: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: Sends 'magic packets' to wake-on-LAN enabled ethernet adapters

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

**Comment by hnyman** [comment]
have you noticed that there is already "etherwake"?
https://github.com/openwrt/packages/blob/master/net/etherwake/Makefile

how is this better?

**Comment by nikil** [comment]
Yes, I'm familiar with etherwake.  etherwake uses TCP packets and requires root.  wakeonlan uses UDP packets and notably doesn't require root.  You can also configure the destination IP and port with wakeonlan (in case you use static ARP tables) which you can't do with etherwake.

**Comment by diizzyy** [comment]
ether-wake is in busybox

File: net/wakeonlan/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline net/wakeonlan/Makefile]: Please don't use OpenWrt.org copyright unless you are under contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

**Comment by nikil** [comment]
Done - thanks for the comments.

**Pull request merged**